install/kubernetes/helm/subcharts/tracing/templates/ingress.yaml

```diff
@@ -17,16 +15,26 @@ metadata:
     {{- end }}
 spec:
   rules:
+{{- if .Values.ingress.hosts }}
     {{- range $host := .Values.ingress.hosts }}
     - host: {{ $host }}
       http:
         paths:
-          - path: /
+          - path: {{ if .Values.contextPath }} {{ .Values.contextPath }} {{ else }} /{{ .Values.provider }} {{ end }}
```

Comment:
If tracing provider is zipkin, the path would be /zipkin, does zipkin support contextpath now??

Reply:
Yes https://github.com/openzipkin/zipkin/pull/1627. It worked for me - but would be good if someone else could also test it out to confirm.

Reply:
@objectiser removing $ in a "range" loop breaks the Helm chart.